ginga/util/iohelper.py

```diff
@@ -18,8 +18,10 @@ def get_fileinfo(filespec, cache_dir='/tmp', download=False):
     """
     Parse a file specification and return information about it.
     """
-    idx = None
-    name_ext = ''
+    # Loads first science extension by default.
+    # This prevents [None] to be loaded instead.
+    idx = ('SCI', 1)
+    name_ext = '[SCI,1]'
```

Comment:
That would be great if you can force `None` to be something meaningful in the name. The other route I was considering is making default "canonical name" a user preference somewhere, which seems unnecessarily complicated.